### PR TITLE
Relax version constraint on rainbow, allow 3

### PR DESCRIPTION
**Pull request by jaredbeck**
There do not appear to be any breaking changes that
should concern rubocop.

https://github.com/sickill/rainbow/blob/master/Changelog.md

-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [x] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Used the same coding conventions as the rest of the project.
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [x] Added tests (N/A)
* [x] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [x] All tests(`rake spec`) are passing.
* [x] The new code doesn't generate RuboCop offenses that are checked by `rake internal_investigation`.
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.
* [x] Updated cop documentation with `rake generate_cops_documentation` (required only when you've added a new cop or changed the configuration/documentation of an existing cop).

[1]: http://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,7 @@
 * [#5077](https://github.com/bbatsov/rubocop/pull/5077): Add new `Rails/CreateTableWithTimestamps` cop. ([@wata727][])
 * Add new `Style/ColonMethodDefinition` cop. ([@rrosenblum][])
 * Add new `Style/ExtendSelf` cop. ([@drenmi][])
+* Relax version constraint on rainbow, allow 3. ([@jaredbeck][])
```
Comment (pocke) [inline CHANGELOG.md]: Is it a new feature?

Comment (bbatsov) [reply CHANGELOG.md]: Good point. Such changes shouldn't be reflected in the changelog, as they're not user-visible.

**Comment by jaredbeck** [comment]
changelog entry removed

**Comment by jaredbeck** [comment]
i can't wait to make a tom clancy joke when rainbow releases version 6

**Pull request merged**